app/views/components/_important-metadata.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+    items = local_assigns[:items] || {}
+    items = items.reject { |k,v| v.nil? }
+    items = items.merge(items) { |k,v| Array(v).join(",").html_safe }
+-%>
+<% if items.any? %>
+  <div class="app-c-important-metadata">
+    <% items.each do |title, definition| %>
+      <p><%= title %>: <strong><%= definition %></strong></p>
```

Comment:
These paragraphs look quite close together. Focussed link states even overlap slightly. Should there be a little more space?

![screen shot 2017-12-05 at 13 04 30](https://user-images.githubusercontent.com/861310/33608642-0bd86ee6-d9bd-11e7-8cf9-93946964ce2f.png)

Reply:
@miaallers, @alextea  can you advise on this please?